src/Serializer/SerializerInterface.php

```diff
@@ -4,12 +4,8 @@
 
 use CultuurNet\SearchV3\ValueObjects\PagedCollection;
 
-/**
- * Provides an interface for search serializer.
- */
 interface SerializerInterface
```

Comment:
I could not select the line but I think we can shorten `\JMS\Serializer\SerializerInterface`

Reply:
You couldn't select it because I never touched it 😄 
I can shorten it but only by using an alias like I did in https://github.com/cultuurnet/search-v3/pull/43/commits/be3d26cdbb1eadf5c7224820b5d5b83b7fac7157, since it conflicts with the `SerializerInterface` in this file

Reply:
Fixed in https://github.com/cultuurnet/search-v3/pull/43/commits/1c9c3ee0d1dfe440e47e25bc9988b54eda0ab341